.gitignore

```diff
@@ -19,3 +19,4 @@ jetty
 .internal_test_app
 # Ignore byebug history
 .byebug_history
+.DS_Store
```

Comment:
Rather than add platform specific ignores here, can you set this up in your local Global Gitignore?

https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer